controllers/auth.js

```diff
@@ -0,0 +1,58 @@
+const User = require('../models/User')
+const bcrypt = require('bcrypt')
+
+const create = async ( req, res ) => {
+    try {
+      
+      let {password} = req.body
+      const {email, avatar, likedEvents, dislikedEvents, blacklistEvents} = req.body
```

Comment:
when a user is register, there won't be any 'events' data in req.body. You should create a new user data object that has the email information, hashed password, and set liked/disliked/blacklist events to empty, for sending to the database